app/assets/javascripts/metrics_config.js

```diff
@@ -2,6 +2,13 @@
 //= require semantic-ui
 
 // metrics api endpoints
+const parameterize = (str) => {
```

Comment:
Just a note, the [HTML5 spec](https://www.w3.org/TR/2011/WD-html5-20110525/elements.html#the-id-attribute) only requires that IDs be unique and not contain spaces. Parametrizing in this manner (stripping special characters) might be more prone to name collisions (which results in funky behavior).

Perhaps not best practice, but jQuery can actually [select IDs with spaces in them](https://stackoverflow.com/questions/6802765/jquery-dealing-with-a-space-in-the-id-attribute). Would that be a simpler approach?